app/Version.hs

```diff
@@ -12,8 +12,15 @@ import Paths_purescript as Paths
 import qualified Development.GitRev as GitRev
 #endif
 
+-- Unfortunately, Cabal doesn't support prerelease identifiers on versions. To
+-- avoid misleading users who run `purs --version`, we manually add the
+-- prerelease identifier here (if any). When releasing a proper version, simply
+-- set this to an empty string.
+prerelease :: String
+prerelease = "-rc2"
```

Comment:
Is the pre-release identifier necessary if we don't intend to publish it to Hackage?

Reply:
I suppose this is useful for debugging purposes. You do not want someone thinking they have the proper release, when they still have an rc.

Reply:
Yeah, exactly